packages/headless/src/controllers/facets/range-facet/headless-range-facet.ts

```diff
@@ -44,15 +44,12 @@ export function buildRangeFacet<
   return {
     ...controller,
 
-    toggleSelect: (selection: RangeFacetValue) =>
-      dispatch(executeToggleRangeFacetSelect({facetId, selection})),
```

Comment:
Just wondering if `executeToggleRangeFacetSelect` can also be removed?

Reply:
executeToggleRangeFacetSelect is reused by both executeToggleNumericFacetSelect / executeToggleDateFacetSelect